molecule/default/molecule.yml

```diff
@@ -18,13 +20,23 @@ platforms:
     pre_build_image: true
 provisioner:
   name: ansible
-  lint:
-    name: ansible-lint
   playbooks:
-    converge: playbook.yml
+    converge: converge.yml
 scenario:
   name: default
+  test_sequence:
```

Comment:
I guess you know what you're doing here - just out of curiosity: what is this changing wrt the default?

Reply:
Copy-pasta, fixing.